### PR TITLE
Participants model

### LINKED ISSUE #31: ✨ creating basic Event participant model
Create Event participant app and register it in the lets-meet/settings.py file
    Create Event participant model in models.py
    Add tests for Event participant model in tests.py

### PR DESCRIPTION
**Pull request by RonHachmon**
- creating participant model
- testing participant model
depends on PR: https://github.com/redhat-beyond/lets-meet/issues/28 and https://github.com/redhat-beyond/lets-meet/pull/27

fix: #31

### CODE REVIEW

**Review by MortarDefender — DISMISSED** [review]
Looks Great, 
Good Job

**Review by YaelD — DISMISSED** [review]
LGTM

File: events/tests.py [hunk 1]
```diff
@@ -1,7 +1,11 @@
 from django.core.exceptions import ValidationError
 from django.utils import timezone
 from .models import Event
+from .models import EventParticipant
 from datetime import datetime
+from users.models import User
+# from users.tests import user0  # noqa: F401; pylint: disable=unused-variable
```
Comment (kobihk) [inline events/tests.py]: if you don't need it please remove it
```suggestion
```

File: events/tests.py [hunk 2]
```diff
@@ -64,3 +77,50 @@ def test_invalidation(title, date_time_start, date_time_end, expected_error):
...
         current_error = error.messages[0]
     assert expected_error == current_error
+
+
+def create_event_participant(event, user, is_creator):
+    return EventParticipant(event_id=event, user_id=user, is_creator=is_creator)
+
+
+@pytest.fixture
+@pytest.mark.usefixtures("user0")
```
Comment (kobihk) [inline events/tests.py]: I think you can get rid of this line because you have user0 in the next line
```suggestion
```

File: events/tests.py [hunk 3]
```diff
@@ -64,3 +77,50 @@ def test_invalidation(title, date_time_start, date_time_end, expected_error):
...
+def event_participant_creator(new_event, user0):  # noqa: F401; pylint: disable=unused-variable
+    return create_event_participant(new_event, user0, True)
+
+
+@pytest.fixture
+@pytest.mark.usefixtures("user0")
```
Comment (kobihk) [inline events/tests.py]: same ^^
```suggestion
```

File: events/tests.py [hunk 4]
```diff
@@ -64,3 +77,50 @@ def test_invalidation(title, date_time_start, date_time_end, expected_error):
...
+def event_participant_not_creator(new_event, user0):  # noqa: F401; pylint: disable=unused-variable
```
Comment (kobihk) [inline events/tests.py]: same ^^
```suggestion
def event_participant_not_creator(new_event, user0):
```

**Review by kobihk — CHANGES_REQUESTED** [review]

**Review by MortarDefender — DISMISSED** [review]
Changes Approved

File: events/tests.py (hunk 1 above)
Comment (RonHachmon) [reply events/tests.py]: Done

File: events/tests.py (hunk 3 above)
Comment (RonHachmon) [reply events/tests.py]: done

File: events/tests.py (hunk 4 above)
Comment (RonHachmon) [reply events/tests.py]: and Done :)

File: events/tests.py (hunk 2 above)
Comment (RonHachmon) [reply events/tests.py]: done

**Review by YaelD — DISMISSED** [review]
LGTM

File: events/tests.py [hunk 5]
```diff
@@ -64,3 +76,48 @@ def test_invalidation(title, date_time_start, date_time_end, expected_error):
...
+    return create_event_participant(new_event, user0, False)
+
+
+@pytest.fixture
+def persist_event_participant(event_participant_creator):
+    event_participant_creator.event_id.save()
+    event_participant_creator.user_id.save()
+    event_participant_creator.save()
+    return event_participant_creator
+
```
Comment (kobihk) [inline events/tests.py:101]: please add a test to make sure we prevent adding the same user+ same event a few times

**Review by kobihk — CHANGES_REQUESTED** [review]

File: events/tests.py (hunk 5 above)
Comment (RonHachmon) [reply events/tests.py:101]: Done

**Review by MortarDefender — DISMISSED** [review]
Changes Approved

**Review by YaelD — DISMISSED** [review]
Looks good!
Greate job

**Review by MortarDefender — APPROVED** [review]

**Review by danninon — APPROVED** [review]
Nicely done.

**Review by YaelD — APPROVED** [review]

**Review by RavidYael — APPROVED** [review]
LGTM

**Review by kobihk — APPROVED** [review]

**Pull request merged**